QueryLib.js

```diff
@@ -0,0 +1,60 @@
+define([], function() {
+  /////////////////////////////////////////////////////////////////////////////
+  //
+  // Private members
+  //
+  /////////////////////////////////////////////////////////////////////////////
+
+  // Set up statistics definition for client-side query
+  // Total popultion of age groups by gender in census tracts
+  const statDefinitions = ["Hosp_beds", "Occupency"].map(function(fieldName) {
```

Comment:
`statDefinitions` is not returned; it is basically a _"private"_ variable, used only by the public method `queryLayerViewAgeStats` and invisible outside of this module. See [Douglas Crockford's article](https://crockford.com/javascript/private.html) for more info.